examples/40_advanced/example_metrics.py

```diff
@@ -79,9 +79,9 @@ def error_wk(solution, prediction, dummy):
 cls.fit(X_train, y_train)
 
 predictions = cls.predict(X_test)
-print("Accuracy score {:g} using {:s}".
-      format(sklearn.metrics.accuracy_score(y_test, predictions),
-             cls.automl_._metric.name))
+score = sklearn.metrics.accuracy_score(y_test, predictions)
```

Comment:
Do we need these intermediate metrics? Isn't the idea of f-strings to circumvent them?

Reply:
Just for legibility sake I thought it better to have as intermediate vars. 

2 of the examples were actually using the wrong scorer as I imagine it was embedded into the long print expression.
I will comment on the diff where it occured

Reply:
Alright.